app/controllers/orgs/lti_configurations_controller.rb

```diff
@@ -35,6 +35,11 @@ def new
       @lti_configuration = LtiConfiguration.new(lms_type: nil)
     end
 
+    def info
+      lms_type = params[:lms_type] || nil
```

Comment:
What is the value of `params[:lms_type]` here in the nil case? We only need `|| nil` if it can be `false`.